fe2-android/app/src/main/java/com/github/dedis/student20_pop/detail/LaoDetailViewModel.java

```diff
@@ -59,7 +64,11 @@
   private final LiveData<String> mCurrentLaoName =
       Transformations.map(mCurrentLao, lao -> lao == null ? "" : lao.getName());
 
-  private final MutableLiveData<Event<EventType>> mNewLaoEventEvent = new MutableLiveData<>();
+  // TODO: Multiple events from Lao may be concatenated using Stream.concat()
+  private final LiveData<List<com.github.dedis.student20_pop.model.event.Event>> mLaoEvents = Transformations
```

Comment:
If I understand correctly, this LiveData will be updated everytime the list of roll calls in the lao is changed, hence making the LaoEvents always up-to-date ? This way, it should be possible to always correctly display the right list of roll calls, right ?